src/GoTrueApi.ts

```diff
@@ -48,6 +48,7 @@ export default class GoTrueApi {
     options: {
       redirectTo?: string
       data?: object
+      hcaptchaToken?: string
```

Comment:
can we update this to just "captcha"? Should it be tied to a specific captcha "type"? If we plan to eventually provide several different security options on gotrue then it might be better to make this param an array/kv of different security options